onnxruntime/core/optimizer/graph_optimizer_registry.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/optimizer/graph_optimizer_registry.h"
+#include "core/optimizer/graph_transformer_utils.h"
+#include "core/optimizer/selection_and_optimization_func.h"
+#include "core/optimizer/qdq_transformer/constant_folding_dq_node.h"
+
+using namespace onnxruntime;
+using namespace ::onnxruntime::common;
+
+namespace onnxruntime {
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
```

Comment:
There doesn't seem to be any practical difference between what happens in a minimal or extended minimal build. neither register any predefined selection funcs. Given that can we simplify and change line 13 to remove the ORT_EXTENDED_MINIMAL_BUILD condition? with that you don't need the ifdef in CreatePredefinedSelectionFuncs